pom.xml

```diff
@@ -143,7 +156,7 @@
         <dependency>
             <groupId>org.jvnet.hudson</groupId>
             <artifactId>xstream</artifactId>
-            <version>1.3.1-hudson-8</version>
+            <version>1.4.7-jenkins-1</version>
```

Comment:
I can submit a partial dependency hell cleanup PR, but I want to keep scope of this change as small as possible. This change just bumps to the the version bundled into the target core

Reply:
Surely you could just delete this dep altogether?

Reply:
Nope, because it would cause upper bound issues with JFrog libs. I would leave it to the maintainers